modules/logging-content-filter-drop-records.adoc

```diff
@@ -0,0 +1,109 @@
+
+.Prerequisites
+
+* You have installed the {clo}.
+* You have administrator permissions.
+* You have created a `ClusterLogForwarder` custom resource (CR).
+
+.Procedure
+
+. Add configuration for a filter to the `filters` spec in the `ClusterLogForwarder` CR.
++
+The following example shows how to configure the `ClusterLogForwarder` CR to drop log records based on regular expressions:
++
+.Example `ClusterLogForwarder` CR
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+# ...
+spec:
+  filters:
+  - name: <filter_name>
+    type: drop # <1>
+    drop: # <2>
+      test: # <3>
+      - field: .kubernetes.labels."foo-bar/baz" # <4>
+        matches: .+ # <5>
+      - field: .kubernetes.pod_name
+        notMatches: "my-pod" # <6>
+  pipelines:
+  - name: <pipeline_name> # <7>
+      filterRefs: ["<filter_name>"]
+# ...
+----
+<1> Specify the type of filter. The `drop` filter completely drops log records that match the filter configuration.
+<2> Specify configuration options for applying the `drop` filter.
+<3> Specify the configuration for tests that are used to evaluate whether a log record is dropped. If all the conditions specified for a test are true, the test passes and the log record is dropped. When multiple tests are specified for the `drop` filter configuration, if any of the tests pass, the record is dropped. If there is an error evaluating a condition, for example, the field is missing from the log record being evaluated, that condition evaluates to false.
+<4> Specify a dot-delimited field path, which is a path to a field in the log record. The path can contain alpha-numeric characters and underscores (`a-zA-Z0-9_`), for example, `.kubernetes.namespace_name`. If segments contain characters outside of this range, the segment must be in quotes, for example, `.kubernetes.labels."foo.bar-bar/baz"`. You can include multiple field paths in a single `test` configuration, but they must all evaluate to true for the test to pass and the `drop` filter to be applied.
+<5> Specify a regular expression. If log records match this regular expression, they are dropped. You can set either the `matches` or `notMatches` condition for a single `field` path, but not both.
+<6> Specify a regular expression. If log records do not match this regular expression, they are dropped. You can set either the `matches` or `notMatches` condition for a single `field` path, but not both.
+<7> Specify the pipeline that the `drop` filter is applied to.
+
+. Apply the `ClusterLogForwarder` CR by running the following command:
++
+.Example command
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+[id="logging-content-filter-drop-records-additional-examples_{context}"]
+== Additional examples
+
+The following additional example shows how you can configure the `drop` filter to only keep important log records:
```

Comment:
I might change this to say "only keep higher priority logs"
You are using the same field for both match and notMatch, which can be accomplished with only the notMatch.   Additionally, in most cases the input filter will be a better way of filtering by namespace (although you can use regex in these tests, which you cannot for namespace filtering).  

A better example may be checking the level or message for higher priority messages:  
```
      - field: .message
        notMatches: "(?i)critical|error"
      - field: .level
        matches: "info|warning"
```

Reply:
>I might change this to say "only keep higher priority logs"

I think this is actually more subjective than "important", since some log information is sort of objectively more important than others? Priority is sort of in the eye of the beholder. Either way, I don't want to spend too much time on semantics at this point, so if it's not a blocker, I'd like to keep this as is.